lib/lib-utils/src/main/java/org/sagebionetworks/util/csv/CSVWriterProviderImpl.java

```diff
@@ -0,0 +1,45 @@
+package org.sagebionetworks.util.csv;
+
+import java.io.Writer;
+
+import org.sagebionetworks.repo.model.table.CsvTableDescriptor;
+
+import au.com.bytecode.opencsv.CSVWriter;
+import au.com.bytecode.opencsv.Constants;
+
+public class CSVWriterProviderImpl implements CSVWriterProvider {
+
+	@Override
+	public CSVWriter createWriter(Writer writer, CsvTableDescriptor csvTableDescriptor) {
```

Comment:
`CSVWriterProvider` and `Impl` were moved from `services-workers` to `lib-utils`. This implementation was moved here from `lib-table-cluster`.